src/views/privacypolicy-apps/l10n.json

```diff
@@ -0,0 +1,68 @@
+{
+    "privacyApps.title":"Privacy Policy",
+    "privacyApps.updated":"The Scratch Privacy Policy was last updated: [DATE]",
```

Comment:
We will need to update "[DATE]" to the actual date when we know when we are going to deploy this.